src/js/services/srv-widget.js

```diff
@@ -233,6 +233,15 @@ app.service('widgetSrv', function(selectedWidgetSrv, dateTimeServ, $q, $rootScop
 		}
 	}
 
+	const canBePublishedByCurrentUser = inst_id => {
```

Comment:
This is implemented 2x in the code - is there any way to use the widget service in the selected widget service to prevent duplication?